app/assets/stylesheets/cart.css.scss

```diff
@@ -9,6 +11,12 @@
   font-size: 42px;
 }
 
+#cart-title{
+  font-family: "Open Sans";
+  font-weight: bold;
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.